internal/querynode/task.go

```diff
@@ -715,6 +715,21 @@ func (r *releaseCollectionTask) releaseReplica(replica ReplicaInterface, replica
 			if err != nil {
 				log.Warn(err.Error())
 			}
+			_, err = r.node.tSafeReplica.getTSafe(channel)
```

Comment:
why do we not move the check into removeTSafeWatcher

Reply:
because we should remove both tSafeWatcher and collection's channel,
how about returning `isRemoved` in removeTSafe?

Reply:
> 

that would make more sense to me~

Reply:
okay